### PR TITLE
RLP user guides

### PR DESCRIPTION
**Pull request by eguzki**

### CODE REVIEW

File: doc/demo-rlp-authenticated.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+  name: key-b
+  labels:
+    authorino.kuadrant.io/managed-by: authorino
+    api: toystore
+stringData:
+  api_key: KEY-B
+type: Opaque
+EOF
+```
+Check `toystotre` HTTPRoute requires API key
```
Comment (rahulanand16nov) [inline doc/demo-rlp-authenticated.md:119]: nitpick: `toystore` is it?

**Review by rahulanand16nov — CHANGES_REQUESTED** [review]

File: doc/demo-rlp-authenticated.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+    name: toystore
+  rules:
+  - to:
+    - operation:
+        hosts: ["api.toystore.com"]
+  action: CUSTOM
+  provider:
+    name: kuadrant-authorization
+EOF
+```
```
[the remark this answers is not in the document]
Comment (eguzki) [reply doc/demo-rlp-authenticated.md:75]: good catch. Fixed

[the remark this answers is not in the document]
Comment (rahulanand16nov) [reply doc/demo-rlp-authenticated.md]: Now, I don't the `AuthPolicy` at all in the example?

[the remark this answers is not in the document]
Comment (eguzki) [reply doc/demo-rlp-authenticated.md]: the demo uses the annotation approach, which is, for now, perfectly valid

[the remark this answers is not in the document]
Comment (rahulanand16nov) [reply doc/demo-rlp-authenticated.md]: ohk, no problem from my side.

**Review by rahulanand16nov — APPROVED** [review]

**Pull request merged**